Apps/Log/Tests iOS/LogIOSLaunchTests.swift

```diff
@@ -1,10 +1,13 @@
+<<<<<<< HEAD:Apps/Log/Tests iOS/LogIOSLaunchTests.swift
 //
 //  LogIOSLaunchTests.swift
 //  Tests iOS
 //
 //  Created by Daniel Tombor on 20/02/2022.
 //
 
+=======
+>>>>>>> 2541e78 (Create Nest workspace):Apps/Log/Tests iOS/Tests_iOSLaunchTests.swift
```

Comment:
!

Reply:
I obviously didn't run the tests for the Log project, busted 😅😂